client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -64,6 +65,29 @@ class Dashboard extends Component {
 		}
 	}
 
+	getBreadcrumb = () => {
+		const {
+			finishedInstallOfRequiredPlugins,
+			finishedInitialSetup,
+			setStoreAddressDuringInitialSetup,
+			translate
+		} = this.props;
+
+		if ( ! finishedInstallOfRequiredPlugins ) {
+			return translate( 'Installing Plugins' );
+		}
+
+		if ( ! setStoreAddressDuringInitialSetup ) {
+			return translate( 'Store Location' );
+		}
+
+		if ( ! finishedInitialSetup ) {
+			return translate( 'Store Setup' );
+		}
+
+		return translate( 'Orders' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **99** times:
`translate( 'Order' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/woo-headers-on-dashboard)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).